account_asset_management/models/account_asset_profile.py

```diff
@@ -169,6 +169,7 @@ class AccountAssetProfile(models.Model):
         "posted depreciation line will prompt the option to reverse the "
         "journal entry, instead of deleting them.",
     )
+    total_number_of_use = fields.Integer(store=True)
```

Comment:
You don't need to add `store=True` because it is not a computed (nor related) field. It is always stored by default.